examples/pylab_examples/figure_title.py

```diff
@@ -3,6 +3,7 @@
 Figure Title
 ============
 
+Example use of `title` and `suptitle` to create subplot titles and a centered figure title, respectively.
```

Comment:
`` not `

also, make sure this doesn't go over 79 characters...not sure if it does or not but it feels a little long to me :-)